### PR TITLE
style: permissions modal

### PR DESCRIPTION
**Pull request by IrakliJani**
Closes: https://github.com/lightdash/lightdash/issues/3817

### Description:

- [x] 1. "Full Access and "Admin" should have the same font-weight ( Admin is the correct one). This is my bad - figma error!
CleanShot 2022-11-25 at 00 02 45@2x

- [x] 2. Members of Thyme to shine Market > Members of 'Thyme to shine Market' with an elipsis(...) if the text is too long (we have preview projects where the project names are really long)

CleanShot 2022-11-25 at 00 18 24@2x

CleanShot 2022-11-25 at 00 13 20@2x

CleanShot 2022-11-25 at 00 17 20@2x

- [x] 3. Get rid of the tick on the left-hand-side of the user's name. If the user is selected, the background colour will be blue.

CleanShot 2022-11-25 at 01 13 45@2x

- [x] 4. Get rid of the popover arrow (the triangle). It can work like a normal dropdown, rather than a popover.

CleanShot 2022-11-25 at 00 45 36@2x

CleanShot 2022-11-25 at 00 45 39@2x

- [x] 5. The Share button should not increase in height when the input increases in height. It should either be pinned to the top, or remain in the middle. Check out Figma for inspo.

CleanShot 2022-11-24 at 23 52 38@2x

- [x] 6. The modal should have a fixed position further up top on the page ( its position should not jump around on the page as it increases/decreases in height)

https://user-images.githubusercontent.com/962095/203865970-c7ef3836-531e-4c52-87ad-3f7688f0cdf2.mp4

- [x] 7.Bottom user's name is not aligned with role.

CleanShot 2022-11-25 at 00 47 37@2x

- [x] 8. Placeholder text colour in the input should be lighter (it should match the colour of the placeholder text in the 'create new space' modal

CleanShot 2022-11-25 at 00 47 02@2x

### CODE REVIEW

File: packages/frontend/src/components/common/ShareSpaceModal/ShareSpaceUserList.tsx [hunk 1]
```diff
@@ -55,11 +55,20 @@ export const ShareSpaceUserList: FC<ShareSpaceUserListProps> = ({
...
     return (
         <>
-            {space.access &&
-                space.access.map((sharedUser) => {
-                    const isYou = sessionUser?.userUuid === sharedUser.userUuid;
+            {space.access
+                .sort((a, b) => {
+                    if (userIsYou(a) && !userIsYou(b)) return -1;
+                    if (!userIsYou(a) && userIsYou(b)) return 1;
+                    return 0;
```
Comment (IrakliJani) [inline packages/frontend/src/components/common/ShareSpaceModal/ShareSpaceUserList.tsx:67]: pins current user to the top

File: packages/frontend/src/App.styles.tsx [hunk 2]
```diff
@@ -1,6 +1,16 @@
+import { Colors } from '@blueprintjs/core';
 import { createGlobalStyle } from 'styled-components';
 
 const AppStyle = createGlobalStyle`
+    .bp4-multi-select-tag-input-input::placeholder {
```
Comment (rephus) [inline packages/frontend/src/App.styles.tsx:5]: this will  affect all multi-selects and dialogs, is this what we want ?

**Comment by rephus** [comment]
Deleting one item, removes all items from the list (this could have been me !) 
[Screencast from 25-11-22 09:49:24.webm](https://user-images.githubusercontent.com/1983672/203939558-331a5e60-de76-4518-962e-138f9f916b6a.webm)

**Comment by rephus** [comment]
You probably need to rebase from master, because I fixed this yesterday (users should not show up when permission is set to public) 
image

File: packages/frontend/src/components/common/ShareSpaceModal/index.tsx [hunk 3]
```diff
@@ -50,45 +52,48 @@ const ShareSpaceModal: FC<ShareSpaceProps> = ({ space, projectUuid }) => {
...
                     ) : null}
+
                     <ShareSpaceAccessType
                         projectUuid={projectUuid}
                         space={space}
                         selectedAccess={selectedAccess}
                         setSelectedAccess={setSelectedAccess}
                     />
+
                     <ShareSpaceUserList
```
Comment (rephus) [inline packages/frontend/src/components/common/ShareSpaceModal/index.tsx:76]: I changed this yesterday to hide the list of users (probably need to rebase)

**Review by rephus — APPROVED** [review]

**Comment by PriPatel** [comment]
Looks great!!! Thank you both!

File: packages/frontend/src/App.styles.tsx (hunk 2 above)
Comment (IrakliJani) [reply packages/frontend/src/App.styles.tsx:5]: good question.. I guess we should change all (including other kinds of inputs from blueprint). 
@PriPatel wdyt? should we change the colour for all input placeholders or just this one. If yes - then im gonna create a separate issue for it.

**Comment by IrakliJani** [comment]
> Deleting one item, removes all items from the list (this could have been me !) [Screencast from 25-11-22 09:49:24.webm](https://user-images.githubusercontent.com/1983672/203939558-331a5e60-de76-4518-962e-138f9f916b6a.webm)

hmm, can't reproduce. let's tackle it as a separate issue?

**Pull request merged**